cv32e40s/sim/tools/README.md

```diff
@@ -1,7 +1,3 @@
 ## TOOLS directory
-Tool-specific sub-dirs for various tools used in the CV32 project.  Please do
-**not** place these in the sim, sim/core or sim/uvmt\_cv32 directories.
-
-#### mcy
-Setup and configuration for [Mutual Coverage with Yosys](https://github.com/YosysHQ/mcy) from [SymbioticEDA](https://www.symbioticeda.com/).
-
+Tool-specific sub-dirs for various tools used in the CV32E40S project.  Please do
+**not** place these in the sim, sim/core or sim/uvmt\_cv32e40s directories.
```

Comment:
Not that it matters (and it doesn't have to be changed), but the corresponding 40p and 40x readmes got the _"Please do [...]"_ sentence starting on its own new line.

Reply:
You are right that it does not matter - the Markdown is rendered the same in both cases - but I made the change anyway.  Just FYI, the reason for the newline is a guideline I read that recommends that each sentence in a Markdown file start and end on its own line.  I do not follow it rigorously, but it does make maintenance of Markdown files easier.

Since I was editing these files I also added a sentence to indicate that the contents of the `tools` directory is not maintained.